sycl/doc/GetStartedGuide.md

```diff
@@ -164,11 +164,15 @@ which contains all the symbols required.
 To run DPC++ applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
+To run DPC++ applications on Level Zero devices, Level Zero implementation(s)
+must be present in the system.
+
 Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel
 runtime versions.
 
-The `GPU` runtime that is needed to run DPC++ application on Intel `GPU` devices
-can be downloaded from the following web pages:
+To run DPC++ application on Intel `GPU` devices the OpenCL `GPU` runtime or the
```

Comment:
#1699 have this wording changed a bit.

Reply:
Thx, according to new wording I just need to specify the version of the L0 runtime in buildbot/dependency.conf. I am not sure if the purpose of this file is just to provide information about version or if it used in some scripts. I will ask people working on CI and then will update this file.

Reply:
buildbot/dependency.conf is used in CI scripts. Added comment to this file explaining that there is a same driver for Level Zero and OpenCL,